orttraining/orttraining/test/graph/bert_toy_fetches.h

```diff
@@ -8,8 +8,9 @@
 static std::unordered_map<std::string, std::vector<double>> BERT_TOY_FETCHES = {
```

Comment:
Include vector?

Reply:
Since this PR address build problem. We should keep the change minimal, so I will leave all those clang-tidy warnings along, and only touch what is needed.